internal/config/defaults.go

```diff
@@ -24,24 +23,31 @@ var (
 
 	// operations holds the supported operations inside mesh
 	operations = map[string]interface{}{
-		installKuma: map[string]interface{}{
-			"type": "INSTALL",
+		InstallKuma: map[string]interface{}{
```

Comment:
Do adapter operations belong in a `defaults` file?

Reply:
@leecalcote Its either inject the config, or have it remained as a default. Injecting would mean that u do it via the `main.go` file. Do we want to have another hop for this?

Reply:
Renaming this file to `supported_operations.go` will probably suffice.

Reply:
out of curiosity, do we have unsupported operations?